src/neo/UInt256.cs

```diff
@@ -25,11 +27,8 @@ public UInt256()
 
         public unsafe UInt256(ReadOnlySpan<byte> value)
         {
-            fixed (ulong* p = &value1)
-            {
-                Span<byte> dst = new Span<byte>(p, Length);
-                value[..Length].CopyTo(dst);
-            }
+            Span<byte> dst = MemoryMarshal.AsBytes(MemoryMarshal.CreateSpan(ref value1, Length / sizeof(ulong)));
+            value[..Length].CopyTo(dst);
         }
```

Comment:
I personally prefer to keep the original code, which can be used in `Uint160`

Reply:
@john-h-k Can we apply this to `UInt160` too? https://github.com/neo-project/neo/blob/master/src/neo/UInt160.cs

Reply:
@john-h-k Could you help merge it [PR#1](https://github.com/john-h-k/neo/pull/1), I have added some changes

Reply:
We should unify the way in both, Uint256 and Uint160

Reply:
Need your commit, we don't want to send another PR. 😂

Reply:
Should this be reverted?

Reply:
I think that the constructor should be equal, which version do you prefer?

Reply:
I think the old one is better.